### PR TITLE
Add support for LEF57_MINSTEP in ROUTING & LEF57_SPACING in CUT

### PR DESCRIPTION
**Pull request by refaay**

### CODE REVIEW

File: src/odb/src/lefin/lefin.cpp [hunk 1]
```diff
@@ -694,7 +694,7 @@ void lefin::layer(lefiLayer* layer)
       } else
         supported = false;
     } else if (type.getValue() == dbTechLayerType::CUT) {
-      if (!strcmp(layer->propName(iii), "LEF58_SPACING")) {
+      if (!strcmp(layer->propName(iii), "LEF58_SPACING") | !strcmp(layer->propName(iii), "LEF57_SPACING")) {
```
Comment (QuantamHD) [inline src/odb/src/lefin/lefin.cpp:697]: Even though bitwise operator `|` is technically correct in this case it's much preferable to use the `||` logical operator when dealing with truth values. Please change to the logical operator `||`

**Comment by QuantamHD** [comment]
You should also include a unit test for this functionality

https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/odb/test/cpp/TestAbstractLef.cc

File: src/odb/src/lefin/lefin.cpp (hunk 1 above)
Comment (refaay) [reply src/odb/src/lefin/lefin.cpp]: Done

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
Please clang-format the code to fix the error

**Comment by maliberty** [comment]
Please add the test as requested

**Review by osamahammad21 — APPROVED** [review]

**Pull request merged**